spec/controllers/invitations_controller_spec.rb

```diff
@@ -143,7 +143,7 @@
   end
 
   describe 'redirect logged out users to the sign in page' do
-    it 'redriects #new' do
+    it 'redirects #new' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.